Source/ACE.Entity/BaseAceObject.cs

```diff
@@ -5,12 +5,60 @@
 
 namespace ACE.Entity
 {
-    [DbTable("base_ace_object")]
-    [DbGetList("base_ace_object", 15, "typeId")]
+    [DbTable("ace_object")]
+    [DbGetList("ace_object", 15, "typeId")]
     public class BaseAceObject
     {
-        [DbField("baseAceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
+        [DbField("AceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
         public virtual uint AceObjectId { get; set; }
+
+        [DbField("bitField", (int)MySqlDbType.UInt32)]
+        public uint AceObjectDescriptionFlags { get; set; }
+
+        [DbField("animationFrameId", (int)MySqlDbType.UInt32)]
+        public uint AnimationFrameId { get; set; }
+
+        [DbField("currentMotionState", (int)MySqlDbType.Text)]
+        public string CurrentMotionState { get; set; }
+
+        [DbField("iconId", (int)MySqlDbType.UInt32)]
+        public uint IconId { get; set; }
+
+        [DbField("iconOverlayId", (int)MySqlDbType.UInt32)]
+        public uint IconOverlayId { get; set; }
+
+        [DbField("iconUnderlayId", (int)MySqlDbType.UInt32)]
+        public uint IconUnderlayId { get; set; }
+        /// <summary>
+        /// PhysicsData.CSetup
+        /// </summary>
+        [DbField("modelTableId", (int)MySqlDbType.UInt32)]
+        public uint ModelTableId { get; set; }
+
+        [DbField("motionTableId", (int)MySqlDbType.UInt32)]
+        public uint MotionTableId { get; set; }
+
+        [DbField("physicsBitField", (int)MySqlDbType.UInt32)]
+        public uint PhysicsDescriptionFlag { get; set; }
+
+        [DbField("physicsScript", (int)MySqlDbType.UInt16)]
+        public ushort PhysicsScript { get; set; }
+
+        [DbField("physicsTableId", (int)MySqlDbType.UInt32)]
+        public uint PhysicsTableId { get; set; }
+
+        [DbField("soundTableId", (int)MySqlDbType.UInt32)]
+        public uint SoundTableId { get; set; }
+
+        [DbField("header", (int)MySqlDbType.UInt32)]
+        public uint WeenieHeaderFlags { get; set; }
+
+        [DbField("spellId", (int)MySqlDbType.UInt16)]
+        public ushort SpellId { get; set; }
```

Comment:
if not needed to make a separate table PropertyInt.ItemSpellcraft _might_ be the corresponding property